source/Editor.js

```diff
@@ -125,17 +125,22 @@ function Squire ( root, config ) {
     this._path = '';
     this._willUpdatePath = false;
 
-    if ( 'onselectionchange' in doc ) {
-      this.addEventListener( 'selectionchange', this._updatePathOnEvent );
-    } else {
+    // nodes.  To do this with the 'selectionchange' event is trickier since our current procedure triggers
```

Comment:
slightly worried we may end up diverging completely from mainline squire, which is almost never a good thing ... but maybe something to reintegrate at a later date, we could ask the squire team about a non-editable strategy they would be happy with